R/utilities.R

```diff
@@ -1725,11 +1725,11 @@ get_gambl_colours = function(classification = "all",
                             "NEG" = "#E5A4CB")
 
   all_colours[["BL"]] = c("M53-BL" = "#A6CEE3",
-                          "DLBCL-1" = "#721F0F",
+                          "DLBCL-A" = "#721F0F",
```

Comment:
Just curious, what's the rationale for moving away from numbers to letters in this case?

Reply:
Ah, this is because we will rename these subgroups as part of our response to reviewer's comments fr our manuscript. The concern is that the clusters C1 to C5 already exist for DLBCL, and this may create confusion